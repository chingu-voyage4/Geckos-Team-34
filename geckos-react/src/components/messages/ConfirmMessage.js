import React from 'react';
import { Message } from 'semantic-ui-react';

const ConfirmMessage = () => {
  return (
    <Message info>
      <Message.Header>Verify your email!</Message.Header>
    </Message>
  );
};

export default ConfirmMessage;
