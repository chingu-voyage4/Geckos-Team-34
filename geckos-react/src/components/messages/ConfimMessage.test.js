import React from 'react';
import { mount } from 'enzyme';

import ConfirmMessage from '../messages/ConfirmMessage';

describe('<ConfirmMessage />', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = mount(<ConfirmMessage />);
  });

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.length).toBe(1);
  });

  it('should render a Message element', () => {
    const elem = wrapper.find('Message');
    expect(elem.length).toBe(1);
  });

  it('should render Message text', () => {
    const elem = wrapper.find('.header');
    expect(elem.length).toBe(1);
    expect(elem.text()).toBe('Verify your email!');
  });
});

