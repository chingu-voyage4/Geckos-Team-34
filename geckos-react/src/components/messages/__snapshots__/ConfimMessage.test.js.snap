// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ConfirmMessage /> should render correctly 1`] = `
<ConfirmMessage>
  <Message
    info={true}
  >
    <div
      className="ui info message"
    >
      <MessageHeader>
        <div
          className="header"
        >
          Verify your email!
        </div>
      </MessageHeader>
    </div>
  </Message>
</ConfirmMessage>
`;
